kvui.py

```diff
@@ -358,39 +358,40 @@ def on_touch_down(self, touch):
         """ Add selection on touch down """
         if super(HintLabel, self).on_touch_down(touch):
             return True
-        alt: bool = touch.button == "right" or touch.is_double_tap
+        # alt: bool = touch.button == "right" or touch.is_double_tap
         if self.collide_point(*touch.pos):
-            if self.index:  # skip header
-                if alt:
-                    if self.hint["status"] == HintStatus.HINT_FOUND:
-                        return
-                    ctx = App.get_running_app().ctx
-                    if ctx.slot == self.hint["receiving_player"]:  # If this player owns this hint
-                        # open a dropdown
-                        self.dropdown.open(self.ids["status"])
-                else:
-                    if self.selected:
-                        self.parent.clear_selection()
+            _parent = self.parent
+            if not self.index:  # clicked on header
+                _parent.clear_selection()
+            parent: HintLog = _parent.parent
+            # find correct column
+            for child in self.children:
+                if child.collide_point(*touch.pos):
+                    key = child.sort_key
+                    if self.index:  # skip header
+                        if key == "status":
```

Comment:
Rather than loop the children, you can just directly get the status label by doing 
```py
status_label = self.ids["status"]
if status_label.collide_point(*touch.pos):
```